### ISSUE TITLE
--phenoExclude to exclude phenotypes from phentoype file

### ISSUE THREAD

**Issue by Shicheng-Guo** [rgcgithub/regenie#185]
It will be useful to have an option `--phenoExclude` to exclude few phenotypes from phentoype file

```
-p, --phenoFile FILE          phenotype file (header required starting with
                                FID IID)
      --phenoCol STRING         phenotype name in header (use for each
                                phenotype to keep; can use parameter expansion
                                {i:j})
      --phenoColList STRING,..,STRING
                                comma separated list of phenotype names to
                                keep (can use parameter expansion {i:j})
```

**Comment by Shicheng-Guo** [comment]
`--phenoExclude` It will be also helpful to avoid 

     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X642).ERROR: logistic regression did not converge for phenotype X646. Perhaps increase --niter?

**Comment by Shicheng-Guo** [comment]
Hi Joelle, Is there any timeline when this enhancement could be released. Thanks.

**Comment by Shicheng-Guo** [comment]
Hi Joelle, 

**"logistic regression did not converge"** is still the most frequency issue for regenie 2.2.4, especially for the region based phewas analysis. Looking forward to having `--phenoExclude` to be developed in next version. 

Thanks. 

Shicheng

```
Chromosome 19 [7 sets in total]
   -reading loco predictions for the chromosome...done (474534ms)
   -fitting null logistic regression on binary phenotypes...
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X071).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X071.1).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X149.3).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X149.4).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X250.22).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X272.13).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X282).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X282.9).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X286.12).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X292.2).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X300.3).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X304).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X346.1).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X350.3).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X356).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X385.5).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X426.4).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X427.61).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X443.7).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X472).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X505).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X513.3).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X513.4).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X520).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X520.2).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X580.31).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X580.32).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X588).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X592.11).
     WARNING: Fitted probabilities numerically 0/1 occured (phenotype #X592.2).ERROR: logistic regression did not converge for phenotype X610.2. Perhaps increase --niter?
(END)
```

**Comment by joellembatchou** [comment]
Hi Shicheng,

Yes, it will be included in the next release (most likely will be out early 2022). 

Cheers,
Joelle

**Comment by joellembatchou** [comment]
Now available in v3.0:  `--phenoExcludeList`
